cli/tools/coverage/mod.rs

```diff
@@ -170,16 +171,16 @@ struct CoverageReport {
 
 fn generate_coverage_report(
   script_coverage: &ScriptCoverage,
-  script_source: &str,
+  script_source: String,
```

Comment:
This is one place we're "regressing" on copies a bit, but given that it's coverage I think it's worth it.